### DISCUSSION TITLE
Lines plugin not working correctly

### DISCUSSION THREAD

**Discussion by revilowaldow** [lowlighter/metrics#1445] [Q&A]
The lines plugin is wrapping incorrectly and leaving large amounts of whitespace.
it used to render correctly.

### To reproduce:
Look at the example:
https://github.com/lowlighter/metrics/blob/examples/metrics.plugin.lines.history.svg
![image](https://github.com/lowlighter/metrics/assets/52298102/f75641af-3dc6-4aa2-9748-f487acd38936)

How it looks on my personal metrics:
![image](https://github.com/lowlighter/metrics/assets/52298102/8f4114d0-cca3-453a-a686-9042787aaa4e)
https://github.com/revilowaldow/revilowaldow/blob/main/metrics.core.svg

How it used to look and how it should look:
![image](https://github.com/lowlighter/metrics/assets/52298102/36b90d51-364c-4a2d-944d-42866fddce20)
https://github.com/revilowaldow/revilowaldow/commit/0ff1f2335e51d7e5ecfb82299e4676dc51b4b9a3

**Comment by revilowaldow** [comment]
Bumping this as it's still happening

**Comment by lowlighter** [comment]
Should be fixed in new release now
